wolfcrypt/src/poly1305.c

```diff
@@ -700,8 +700,6 @@ int wc_Poly1305Update(Poly1305* ctx, const byte* m, word32 bytes)
     }
     printf("\n");
 #endif
-    if (ctx == NULL || (m == NULL && bytes > 0))
```

Comment:
Why remove the `ctx` and case where bytes is provided and point is null check?

Reply:
The test on L703 was a duplicate -- the same test was and remains applied at line 686 (func entry).